src/text_engine/interaction/Interactive.java

```diff
@@ -6,15 +6,37 @@
 
 import text_engine.boundaries.Room;
 import text_engine.characters.GameCharacter;
+import text_engine.characters.PlayerCharacter;
 import text_engine.constants.Prompts;
+import text_engine.interaction.actions.ActionList;
+import text_engine.interaction.actions.BaseAction;
 import text_engine.items.GameEntity;
 
 public abstract class Interactive {
 
     /**
+     * Get the base set of actions to present to the player. Should be fetched and then added to by
+     * each {@link Interactive} subclass for context-specific options.
+     *
+     * @param actor  {@link GameCharacter} interacting with {@link this}
+     * @param from   {@link GameEntity} that the {@link GameCharacter} came from
+     * @param reader to read response from
+     * @param writer to print prompt to
+     * @return the base set of player actions
+     */
+    protected ActionList actions(
+            GameCharacter actor, GameEntity from, BufferedReader reader, PrintWriter writer) {
+        ActionList actions = new ActionList();
+
+        actions.add(new BaseAction(Prompts.messages.getString("Actions.BACK.name"), "",
+                                   playerCharacter -> (Interactive) from));
```

Comment:
I don't think the global actions list is going to expand beyond a basic "back" option.